tools/build-version.go

```diff
@@ -22,26 +23,37 @@ func getTag(match ...string) (string, *semver.PRVersion) {
 			if ahead, err := semver.NewPRVersion(tagParts[1]); err == nil {
 				return tagParts[0], &ahead
 			}
+			log.Println("semver.NewPRVersion:", err)
```

Comment:
These don't look very useful as is. Are you gonna remove them, or improve them?

Reply:
`semver.NewPRVersion` can return with an error, so I will print it in this case too. In case it's regarding the ambiguity, then we could add something to identify the path. In case not, then please provide a better alternative, before we end up in a ping-pong once again. :wink: